src/domains/chart/components/lib-charts/alarmBadge.js

```diff
@@ -0,0 +1,54 @@
+import React, { forwardRef } from "react"
+import styled from "styled-components"
+
+export const getBackgroundColor = (status) => ({
+  WARNING: "#FFF8E1",
+  CRITICAL: "#FFEBEF",
+  CLEAR: "#E5F5E8",
+})[status] || null
```

Comment:
Since most of the colors does not exist in the pallet can we give it some time and discuss with the designers about that? We could either reuse a color or add those colors in the pallet.

Reply:
sure, i agree, but for now i think we need to stay with those overrides. This is actually why i put those colours here... Because they are not solid and perhaps will change.